user/overriding-packages.rst

```diff
@@ -0,0 +1,283 @@
+Overriding Packages
+===================
+
+Sometimes it's desirable to change the version of a package in a workspace after it's been built without rebuilding the workspace.
+This is called **overriding** the package.
+It's accomplished by sourcing the existing workspace, then building a different version of that package again in a new workspace.
+The existing workspace is called an **underlay workspace**, and the new one is an **overlay workspace**.
+If there are multiple existing workspaces then each one sourced is said to **overlay** the previous one.
+
+Overriding a package is not always possible.
+This page offers tips for avoiding most issues.
+
+.. contents:: Table of Contents
+    :depth: 3
+
+
+How to avoid most issues
+------------------------
+
+These are good practices to avoid most issues.
+If the advice is too restrictive then see the known issue descriptions for more complex advice.
+
+Use isolated workspaces
+***********************
+
+If you are building a workspace and suspect you may overriding a package from it in the future, then use an isolated workspace.
```

Comment:
The reader may not know what an isolated workspace is, so a one-line description would help here.

Reply:
be668ea links to a new page added by this PR that explains what isolated and merged workspaces are.

Reply:
2fe99b7 moves the link to the right spot.

Reply:
```suggestion
If you are building a workspace and suspect you may override a package from it in the future, then use an isolated workspace.
```